nutrition_labels/grant_data_processing.py

```diff
@@ -107,8 +107,7 @@ def only_text(string):
 # getting control data from wellcome grant data
 # 3== not that relevent 4==not at all relevent
 needed_cols = ['Internal ID','Title','Description','tool relevent ']
-grant_code = grant_tags.loc[(grant_tags['tool relevent ']==4.0) | (grant_tags['tool relevent ']==3.0)]
-grant_code['tool relevent '] = 4
+grant_code = grant_tags.loc[grant_tags['tool relevent ']==5.0,needed_cols]
```

Comment:
How come you don't include the other tags you made on the grant_tags data? I think youve explained before, but i've forgotten!

Reply:
Because I didn't tag tool, model and datasets separately, it was just relevant and sort of relevant. Also as they were the first ones I tagged I'm not sure if the tags are that representative of what we want to find

Reply:
👍 thanks